subscription/src/main/java/org/killbill/billing/subscription/api/user/DefaultSubscriptionBase.java

```diff
@@ -381,7 +381,7 @@ public Plan getLastActivePlan() {
 
     @Override
     public PlanPhase getLastActivePhase() {
-        if (getState() == EntitlementState.CANCELLED) {
+        if (getState() == EntitlementState.CANCELLED || getState() == EntitlementState.EXPIRED) { //TODO_1533 added this to prevent tests in TestCatalogFixedTerm from failing after notification processing changes, revisit later
```

Comment:
Same remark here: I think this is correct, you can remove the TODO_1533.

Reply:
Fixed